rewrite-test/src/test/kotlin/org/openrewrite/config/EnvironmentTest.kt

```diff
@@ -318,4 +326,58 @@ class EnvironmentTest : RecipeTest<PlainText> {
             .hasSize(1)
         assertThat(helloJon2.recipeList.first().name).isEqualTo("org.openrewrite.HelloJon")
     }
+
+    @Issue("https://github.com/openrewrite/rewrite/issues/1596")
+    @Test
+    fun removeExclusion() {
+        val env = Environment.builder()
+            .load(
+                YamlResourceLoader(
+                    //language=yaml
+                    """
+                        type: specs.openrewrite.org/v1beta/recipe
+                        name: test.Parent
+                        displayName: Text migration
+                        recipeList:
+                            - test.ParentChild
+                        excludedRecipeList:
+                            - org.openrewrite.text.ChangeText:
+                                toText: Hello
+                        ---
```

Comment:
While excluding, will we still need to provide the parameters even if we just want to exclude them altogether?  Or this is just an example?

Reply:
> While excluding, will we still need to provide the parameters even if we just want to exclude them altogether? Or this is just an example?

The PR is still a draft and is subject to change. But to answer your question, currently, the parameters would be required.

The decision was because recipes like `ChangeType` are relatively common in declarative recipes.
We have declarative recipes like migrations that include other declarative recipes (children). The children recipes may include identical transformations, so multiple instances of the specified exclusion may exist. To prevent the excluded change, each of the children need to exclude the recipe as well.

I.E. `ChangeType` is commonly used.
An example of ChangeType X to Y (ChangeType3:

```yaml
...
name: DeclarativeRecipe1
recipeList:
  - ChangeType1
  - ChangeType2
  - DeclarativeRecipe2
  ...
excludedRecipeList:
  - ChangeType3
```
```yaml
...
name: DeclarativeRecipe2
recipeList:
  - ChangeType3
  - ChangeType4
  - DeclarativeRecipe3
  ...
```
```yaml
name: DeclarativeRecipe3
recipeList:
  - ChangeType3
  - ChangeType5
  ...
```

The PR is a draft to get input and will likely change. We might not add a list of exclusion, or only apply them to the recipe that declares the exclusion, or match by `String` and remove all recipes of that type.